src/core/libs/supl/CMakeLists.txt

```diff
@@ -12,15 +12,21 @@ file(GLOB ASN_SUPL_SOURCES "${CMAKE_SOURCE_DIR}/src/core/libs/supl/asn-supl/*.c"
 list(SORT ASN_SUPL_SOURCES)
 file(GLOB ASN_SUPL_HEADERS "${CMAKE_SOURCE_DIR}/src/core/libs/supl/asn-supl/*.h")
 list(SORT ASN_SUPL_HEADERS)
+file(GLOB ASN_TYPES_SOURCES "${CMAKE_SOURCE_DIR}/src/core/libs/supl/types/*.c")
```

Comment:
As a general rule, CMake recommends _against_ using **glob** patterns. The one place where they are appropriate is with generated files. As @carlesfernandez kindly pointed out, this code was originally generated, which explains the pattern here.

If we are now moving to explicit maintenance, would it be better to explicitly enumerate the sources and their headers? I don't feel strongly about this, but hope that the maintainers will have an opinion.

Reply:
I'd rather leave as is for now if that's ok. It would be a very long list to write out.